nilearn/plotting/surf_plotting.py

```diff
@@ -422,6 +426,248 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
     return display
 
 
+def _check_hemispheres(hemispheres):
+    """Checks whether the hemisphere in plot_img_on_surf is correct.
+
+    hemisphere: list
+        Any combination of 'left' and 'right'
+    """
+    if not isinstance(hemispheres, list):
+        hemispheres = [hemispheres]
+    invalid_hemi = any([hemi not in VALID_HEMISPHERES for hemi in hemispheres])
+    if invalid_hemi:
+        supported = "Supported hemispheres:\n" + str(VALID_HEMISPHERES)
+        raise ValueError("Invalid hemispheres definition!\n" + supported)
+    return hemispheres
+
+
+def _check_views(views) -> list:
+    if not isinstance(views, list):
```

Comment:
same thing for views: let us only accept sequences and avoid the type comparison and conversion to list

Reply:
Done.